src/minoversikt/lagre-filter-controller.tsx

```diff
@@ -0,0 +1,35 @@
+import {useEffect} from "react";
+import {erObjektValuesTomt, lagredeFilterListerErLik} from "../components/modal/lagrede-filter/lagrede-filter-utils";
+import {avmarkerSisteVelgtFilter, avmarkerVelgtFilter, markerVelgtFilter} from "../ducks/lagret-filter";
+import {logEvent} from "../utils/frontend-logger";
+import {finnSideNavn} from "../middleware/metrics-middleware";
+import {useDispatch, useSelector} from "react-redux";
+import {AppState} from "../reducer";
+
+
+export function LagreFilterController(){
```

Comment:
this should be a custom effect ??? i.e useLagreFilterController()... and remove <LagreFilterController/>